hyp3_sdk/hyp3.py

```diff
@@ -13,21 +16,20 @@
 
 class HyP3:
     """A python wrapper around the HyP3 API"""
-    def __init__(self, api_url: str = HYP3_PROD, authenticated_session: Optional[requests.Session] = None):
-        """
 
+    def __init__(self, api_url: str = HYP3_PROD, username: Optional = None, password: Optional = None):
+        """
         Args:
             api_url: Address of the HyP3 API
-            authenticated_session: An authenticated Earthdata Login session to use
+            username: Username for authenticating to urs.earthdata.nasa.gov. Both username and password must be provided
+            password: Password for authenticating to urs.earthdata.nasa.gov. Both username and password must be provided
```

Comment:
`must be provided` contradicts `Optional`, should say something like `if one is provided both must be provided`

Reply:
or accept credentials as a tuple the same way `requests` does: `auth=(username, password)`